dotnet/src/webdriver/Internal/Logging/LogContext.cs

```diff
@@ -60,19 +60,16 @@ public ILogContext CreateContext(LogEventLevel minimumLevel)
                 loggers = new ConcurrentDictionary<Type, ILogger>(_loggers.Select(l => new KeyValuePair<Type, ILogger>(l.Key, new Logger(l.Value.Issuer, minimumLevel))));
             }
 
-            IList<ILogHandler> handlers = null;
+            var context = new LogContext(minimumLevel, this, loggers, null);
```

Comment:
Yes, this is weak place in design. In ctor want to take IList and always create new List. We should always create new list (inherited from parent), because of we want to modify this list and don't affect parent's. I will look at how to make it more confident.

Reply:
Resolved by https://github.com/SeleniumHQ/selenium/commit/00b579d6e3254e282315bd40e86e37a28acf769e